smac/runhistory/runhistory2epm.py

```diff
@@ -461,6 +461,7 @@ def transform_response_values(self, values: np.ndarray) -> np.ndarray:
         """
 
         min_y = self.min_y - (self.perc - self.min_y)  # Subtract the difference between the percentile and the minimum
+        min_y -= 1e-5  # Minimal value to avoid numerical issues in the log scaling below
```

Comment:
This implies that we are always at least 1e-5 away from the optimum. Therefore, it could be harder to hit the optimum with higher precision. In practice, this might not matter; however in our BORF experiments, we always wanted to achieve a regret of 1e-6 (which is of course also an arbitrary number).
In the end, I wonder whether 1e-5 is a reasonable gap to be used here.

Reply:
I'd be happy to change this to 1e-10 or something else which is small enough. On the other hand I was wondering why you want to use a scaled version on the first place.

Reply:
Yes, please change it to 1e-10.
We can discuss the scaled version offline if you want.